addons/mail/static/src/models/follower_list_menu_view.js

```diff
@@ -11,5 +11,11 @@ registerModel({
             inverse: 'followerListMenuView',
             readonly: true,
         }),
+        /**
+         * Determine whether the dropdown is open or not.
```

Comment:
→ Determine**s**?
Actually I don't think this field requires a comment at all.

Reply:
Changes done.